lib/VOMSLibrary.py

```diff
@@ -43,6 +44,30 @@ def compare_dates(self, date0, date1):
 		else:
 			return 1
 
+	def parse_vomses_files(self, vomses_dirs: list = ["/etc/vomses", "~/.glite/vomses"]) -> dict:
+		"""Read the files in /etc/vomses and in ~/.glite/vomses, returning a dictionary with the following items:
+		vo: [hosts]"""
+
+		vomses_files = []
+		for vomses_dir in vomses_dirs:
+			if os.path.exists(os.path.expanduser(vomses_dir)):
+				vomses_files.extend(glob.glob(os.path.expanduser(os.path.join(vomses_dir,"*"))))
+		vomses = {}
+		
+		for vomses_file in vomses_files:
+			try:
+				with open(vomses_file) as f:
+					for line in f:
+						if not line.startswith("#"):
```

Comment:
I would consider a line commented out even if the `#` is not the first character, although this is the current behavior at least of the C/C++ library

Reply:
I can split the line at the ``#`` character and ignore the part after that, but still consider the first part. Do you agree?